src/components/Markdown.svelte

```diff
@@ -17,13 +17,29 @@
   use({ renderer });
```

Comment:
I think we need to enhance the above renderer object to "escape" angle brackets found in HTML tags, as I don't think we want to allow people to specify HTML in their markdown descriptions.